src/main/java/com/google/cloud/spanner/pgadapter/statements/IntermediateStatement.java

```diff
@@ -170,8 +171,8 @@ public void setHasMoreData(boolean hasMoreData) {
     this.hasMoreData = hasMoreData;
   }
 
-  public Statement getStatement() {
```

Comment:
We don't create and hold on to a reference to a `Statement` anymore, as Cloud Spanner `Statement` instances are immutable and can only be used to execute one SQL string with one set of parameters. JDBC statements can be re-used to execute different SQL strings.